scripts/docs/metalsmith/content/concepts.md

```diff
@@ -33,7 +34,19 @@ Taking the first schema for reference, the updates look like that:
 
 ## Managed parameters
 
-Most of the parameters on the Algolia Rest API are simple, a string, a number.
+Idea:
+---
+We need to name drop some managed parameters and non managed parameters in this
+section. To me it was hard to follow because ultimately what we want to say that all
+filtering features of the Algolia Engine (numeric, facets) are string or array string based
+and thus needs to be managed to allow easy manipulation like add, remove, toggle filter
+removing the need for the user to handle it himself.
+
+We can even give an example of clicking a facet value in a list with and without the helper.
+---
+
+Most of the parameters on the Algolia Rest API are simple: you just
+provide a string(?), a number (hitsPerPage).
```

Comment:
Ah yes in the end, query works well too. I don't even know what is queryType for